### ISSUE TITLE
#request QtCreator 4.8.0 & Qt5.12 LTS 

### ISSUE THREAD

**Issue by gbschenkel** [Alexpux/MINGW-packages#4607]
Hi, only notifying the QtCreator IDE have a new release, 4.7.2.
Thanks for all hard work.

**Comment by gbschenkel** [comment]
Hi @Alexpux, sorry about the delay. The last time I tried compile the QtCreator I got a error and couldn't pass it, then I made this request. Well I got my workstation formatted by my employer, and I will try again after I update the MSYS2. Was released today the QtCreator 4.8.0, also the Qt 5.12, which is a LTS version. I was thinking we could have a LTS version of Qt inside MSYS2 which could live together with a current version of Qt. Right now, of course, the last version is the LTS version, but in March, maybe, Qt will be releasing the 6.0 version. I don't know which changes will be need, and if is possible too.
